docs/examples/1d-annotations.html

```diff
@@ -378,5 +378,11 @@
 window.hgApi = window.hglib.viewer(
   document.getElementById('demo'), viewConfig, { bounded: true },
 );
+
+window.hgApi.on(
+  'click', ({ type, event, payload }) => {
+    console.log('Clicked on ', type, '\n', event, '\n', payload);
```

Comment:
This might be the one case where I'd prefer an alert, or else add text to explain that the console needs to be opened to see the result of the event. (But I think we generally could do better at explaining what our demos are supposed to demonstrate, and this is just one instance.)

Reply:
I agree but it depends on the purpose of the examples. I started those simply to have a place for developers to see how certain things can be used. Anyone who knows what a click event listener is should be able to look at the code and understand what's going on. The concept is similar to http://regl.party/examples where the code is simultaneously shown next to the visual output. Something I think would be awesome to have in general.